include/grpc/grpc.h

```diff
@@ -440,6 +440,12 @@ GRPCAPI void grpc_server_set_config_fetcher(
 GRPCAPI int grpc_server_add_insecure_http2_port(grpc_server* server,
                                                 const char* addr);
 
+/** EXPERIMENTAL.  Add a "binder port" to the server that listens to incoming
+ *  binder transactions.
+ *  Returns 1 on success, 0 on failure (or called in non-Android environments)
+ *  REQUIRES: server not started */
+GRPCAPI int grpc_server_add_binder_port(grpc_server* server, const char* addr);
```

Comment:
Public API additions will need a gRFC written.

Reply:
I wonder if we still need this C API for us to wrap C++ API around it. According to https://github.com/grpc/grpc/projects/8 it seems no longer relevant.

Reply:
Whilst I think it's probably a worthwhile idea, I don't think it's an actual official stance for the project (and if it were I think a gRFC would be needed for that).

We need a C API, and we need a gRFC for it.

Reply:
Sure, we will include the this C API into our gRFC.